doc/source/serve/architecture.rst

```diff
@@ -43,11 +44,10 @@ When an HTTP request is sent to the router, the follow things happen:
   are more than ``max_concurrent_queries`` requests outstanding), the request
   is left in the queue until an outstanding request is finished.
 
-Each replica maintains a queue of requests and processes one batch of requests at
-a time. By default the batch size is 1, you can increase the batch size <ref> to
-increase throughput. If the handler (the function for the backend or
-``__call__``) is ``async``, the replica will not wait for the handler to run;
-otherwise, the replica will block until the handler returns.
+Each replica maintains a queue of requests and one at a time, possibly using
```

Comment:
This sentence feels like it's missing some words, but I couldn't figure out exactly how to fix it

Reply:
oops....... definitely removed a couple of words :)